python/ray/util/client/logsclient.py

```diff
@@ -60,6 +72,35 @@ def _log_main(self) -> None:
                 logger.exception(
                     f"Got Error from logger channel -- shutting down: {e}")
 
+    def _keepalive_main(self) -> None:
```

Comment:
This looks similar to the previous main with some differences in warning/wait time? can we unify the two functions?